ChangeLog.md

```diff
@@ -11,6 +11,10 @@
     (Roland Denis, [#359](https://github.com/DGtal-team/DGtalTools/pull/359/files))
   - Using SourceForge to download doxygen sources during Travis CI jobs.
     (Roland Denis [#360](https://github.com/DGtal-team/DGtalTools/pull/360))
+  - volAddBorder: add an option to fill the border of interior volume (Bertrand
```

Comment:
I don't understand what it means..

Reply:
in other words, it changes the image values of voxels located on the frontier of the domain.

Reply:
Is that :

Add an option that sets zero value to domain boundary voxels without changing the domain extent 

?

Reply:
yes exactly, thanks I add sorry for the not clear description

Reply:
thanks